### ISSUE TITLE
Dynamic Data is Mis

### ISSUE THREAD

**Issue by AndrewCarterBreakwaterIT** [SharePoint/sp-dev-fx-webparts#850]
- [x] Question
- [ ] Bug
- [ ] Enhancement

Trying to implement Dynamic Data solution similar to the Dynamic Event example..

I think I have the Source set up correctly, populates from the SP List in a Dropdown. However, when I add the consumer webpart the console starts reporting errors.. 

sp-loader-assembly_default.js:9758 [1556004484242][OtherGlobalError.window.onerro] [SPWebPartErrorCode.SerializationFailed]:
Serialization failed for web part WebPart.TimelineWebPart.b6a3dc31-1248-41b7-b508-af968a5eab3d.

***INNERERROR:
Dynamic Data is misconfigured
***CALLSTACK:
Error
    at SPWebPartError.SPError [as constructor] (https://localhost:4321/temp/workbench-packages/@microsoft_sp-loader/dist/sp-loader-assembly_default.js:8671:24)
    at new SPWebPartError (https://localhost:4321/node_modules/@microsoft/sp-webpart-base/dist/sp-webpart-base_en-us.js:956:28)
    at Function.SPWebPartError.create (https://localhost:4321/node_modules/@microsoft/sp-webpart-base/dist/sp-webpart-base_en-us.js:982:16)
    at TimelineWebPart.BaseWebPart._internalSerialize (https://localhost:4321/node_modules/@microsoft/sp-webpart-base/dist/sp-webpart-base_en-us.js:1218:42)
    at TimelineWebPart.BaseWebPart._internalSetDirtyBit (https://localhost:4321/node_modules/@microsoft/sp-webpart-base/dist/sp-webpart-base_en-us.js:1291:36)
    at https://localhost:4321/node_modules/@microsoft/sp-webpart-base/dist/sp-webpart-base_en-us.js:3706:28
    at https://localhost:4321/node_modules/@microsoft/sp-webpart-base/dist/sp-webpart-base_en-us.js:3669:21
    at Map.forEach (<anonymous>)
    at ClientSideWebPartManager._executeForIdsOrAll (https://localhost:4321/node_modules/@microsoft/sp-webpart-base/dist/sp-webpart-base_en-us.js:3667:28)
    at https://localhost:4321/node_modules/@microsoft/sp-webpart-base/dist/sp-webpart-base_en-us.js:3704:23

The property pane / delete / move options are all disabled and the only option is to delete the section to remove the webpart.

**Comment by AndrewCarterBreakwaterIT** [comment]
I have managed to get further along with this, I think this related to the Dynamic Property not being defined in the Manifest as I have added it there and the webpart errors, but it is a different error.. no getting:

Something went wrong
If the problem persists, contact the site administrator and give them the information in Technical Details.
TECHNICAL DETAILS
[SPLoaderError.loadComponentError]:
***Failed to load component "6861833a-d066-4af2-99d6-a822e4e4c13a" (TimelineWebPart).
Original error: ***Failed to load entry point from component "6861833a-d066-4af2-99d6-a822e4e4c13a" (TimelineWebPart).
Original error: Error loading https://component-id.invalid/6861833a-d066-4af2-99d6-a822e4e4c13a_0.0.1
	Object prototype may only be an Object or null: undefined
